presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -191,6 +176,63 @@ Property Name                                      Description
+Property Name                                      Description                                                  Default
+================================================== ============================================================ ============
+``hive.metastore.uri``                             The URI(s) of the Hive metastore to connect to using the
+                                                   Thrift protocol. If multiple URIs are provided, the first
+                                                   URI is used by default and the rest of the URIs are
+                                                   fallback metastores. This property is required.
+                                                   Example: ``thrift://192.0.2.3:9083`` or
+                                                   ``thrift://192.0.2.3:9083,thrift://192.0.2.4:9083``
+
+``hive.metastore.username``                        The username Presto will use to access the Hive metastore.
+
+``hive.metastore.authentication.type``             Hive metastore authentication type.                          ``NONE``
+                                                   Possible values are ``NONE`` or ``KERBEROS``.
+
+``hive.metastore.service.principal``               The Kerberos principal of the Hive metastore service.
+
+``hive.metastore.client.principal``                The Kerberos principal that Presto will use when connecting
+                                                   to the Hive metastore service.
+
+``hive.metastore.client.keytab``                   Hive metastore client keytab location.
+================================================== ============================================================ ============
+
+AWS Glue Catalog Configuration Properties
+-----------------------------------------
+
+==================================================== ============================================================ ============
+Property Name                                        Description                                                  Default
+==================================================== ============================================================ ============
+``hive.metastore.glue.region``                       AWS Region in which the Glue Catalog to use as metadata
+                                                     provider resides in.
+                                                     Example: ``us-east-1``
+
+``hive.metastore.glue.pin-client-to-current-region`` Boolean indicating if the Glue client should be pinned to
+                                                     the current EC2 region.
+
+``hive.metastore.glue.max-connections``              Max number of concurrent connections to Glue                 ``5``
+
+``hive.metastore.glue.default-warehouse-dir``        Hive Glue metastore default warehouse directory
+
+``hive.metastore.glue.aws-access-key``               AWS access key to use to connect to the Glue Catalog to
```

Comment:
Let's remove the "to use as metadata provider" part, as that's a more general description of Glue and not related to this specific property.